browser-extension/plugin/manifest.firefox.json

```diff
@@ -4,21 +4,20 @@
     "description": "Moderate your Twitter Feed",
     "version": "0.1.10",
     "author": "tattlemade|cis",
-    "content_security_policy": "default-src 'none'; object-src 'none'; script-src 'unsafe-inline' ; font-src https://fonts.gstatic.com; style-src 'unsafe-inline';",
+    "content_security_policy": "default-src 'none'; connect-src http://localhost:3000 https://ogbv-plugin.tattle.co.in/ https://uli-media.tattle.co.in/; font-src https://fonts.gstatic.com; object-src 'none'; script-src 'self' ; style-src https://fonts.googleapis.com 'self' 'unsafe-inline'; img-src https://uli-media.tattle.co.in/; base-uri 'none'; form-action 'none'; frame-ancestors 'none'; report-uri 'none';",
     "permissions": [
         "storage",
         "webRequest",
+        "https://*/*",
```

Comment:
@duggalsu can you review if this needs to be in the permissions array? I remember it getting flagged by the chrome store review team as being too excessive.

Reply:
Confirmed that it is not needed and removed from chrome manifest too